<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AeroChinquihue</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>650</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Servicio:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="flight_button">
        <property name="text">
         <string>Vuelo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="freight_button">
        <property name="text">
         <string>Encomienda</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="name">
      <property name="text">
       <string>ej: Jhon Doe</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Rut:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="identification">
      <property name="text">
       <string>ej:12345678-9</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Destino:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="destination"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="date_label">
          <property name="text">
           <string>Fecha de ida:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCalendarWidget" name="date"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Layout_java">
        <item>
         <widget class="QLabel" name="time_label">
          <property name="text">
           <string>Hora de ida:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="time"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="airplane_label">
      <property name="text">
       <string>Avi√≥n:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="airplane"/>
    </item>
    <item>
     <widget class="QLabel" name="seats_label">
      <property name="text">
       <string>Asientos:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="seats"/>
    </item>
    <item>
     <widget class="QLabel" name="weight_label">
      <property name="text">
       <string>Peso (En Kilogramos)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="weight">
      <property name="text">
       <string>ej:10</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Medio de pago:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="payment_method"/>
    </item>
    <item>
     <widget class="QPushButton" name="ok_button">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
